draft-ietf-httpbis-binary-message.md

```diff
@@ -264,10 +262,9 @@ Request Control Data {
 
 ## Response Control Data
 
-The control data for a request message includes a single field that corresponds
-to the `:status` pseudo-header field in HTTP/2; see {{Section 8.3.2 of
-H2}}. This field is encoded as a single variable length integer, not a decimal
-string.
+The control data for a request message includes a single field that includes the
+status code.  This field is encoded as a single variable length integer, not a
+decimal string.
```

Comment:
For consistency with the request para (and fixing incorrect use of `request message` here)

```suggestion
Response Control Data contains the status code of a response message. It is
encoded as a single variable length integer, not a decimal string.
```